libs/types-common-journal/proto/UserEvents.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto2";
+
+option java_package = "com.wire.events";
+option java_outer_classname = "UserEvents";
+
+message UserEvent {
+    required EventType event_type = 1;
+    required bytes user_id = 2;
+    required int64 utc_time = 3;
+    optional bytes email = 4;
+    optional bytes locale = 5;
+
+    enum EventType {
+        USER_ACTIVATE     = 0;
+        USER_DELETE       = 1;
+        USER_UPDATE       = 2;
+        // --^ When used, the new values for optional fields are
+        //     whatever is set (email, locale) unless they are 
+        //     null / not set (protobuf does not distinguish)
+        USER_EMAIL_REMOVE = 3;
+        // --^ When an email is removed, the email field is set
+        //     to the value of the old email address
```

Comment:
> the email field is set to the value of the old email address

User A registers with email; then adds a phone number, then removes his email (all valid operations w.r.t the API). There would be no "old value" for anyone to set. This comment is not very clear.

Reply:
In that case, the value of the old email (the one that the user just removed) will be on the event https://github.com/wireapp/wire-server/blob/develop/services/brig/src/Brig/API/User.hs#L416

Reply:
what about `when an email address X is removed, the email field is set to X`?

Reply:
> what about when an email address X is removed, the email field is set to X?

👍